api/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Cluster Registry",
-   "version": "v0.0.1"
```

Comment:
Can you make this commit first in the commit tree? I think it would be cleaner if this is the first thing done after the release.

Reply:
Also, my apologies for forgetting to do this last week!

Reply:
I know it's not apparent, but it actually already is as I made sure to do that for the same reasons you suggest. Strangely enough, the commits aren't ordered correctly in the commit history through github's web UI. But if you click to `Browse the repository at this point in the history` on any or all commits (last commit is `Fix gofmt errors`), then click on commits, you'll see the history shows it's the first commit. Perhaps this is because of the tide pool error.

Reply:
Huh. Oh well, thanks for being on top of things. :)